requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy
+sncosmo
```

Comment:
can you specify the minimum version number to make sure that the environment is set up well even if people have old version of sncosmo?

Reply:
Updated